### ISSUE TITLE
Details component closing accessibility

### ISSUE THREAD

**Issue by bameyrick** [ONSdigital/design-system#250]
When a user closes the details component using the inner button, the summary element should be refocused for screen readers to be able to easily move on to the next item on the page.

_Originally posted by @bameyrick in https://github.com/ONSdigital/design-system/issues/148#issuecomment-467793073_

**Comment by bameyrick** [comment]
I haven't been able to recreate this issue, so it appears to have been fixed elsewhere.
